Profiles - ZIB 2017/zib-MedicationAdministration.xml

```diff
@@ -261,7 +261,7 @@
       <path value="MedicationAdministration.category.coding" />
       <sliceName value="MedicationAdministrationCode" />
       <short value="MedicationAdministrationCode" />
-      <definition value="Category snomed code used for indexing/retrieval of the ZIB MedicationAdministration. The SNOMED CT code 422037009 for this resource is stated in the Medication Process v09." />
+      <definition value="Category SNOMED code used for indexing/retrieval of the HCIM MedicationAdministration. The SNOMED code 18629005 for this resource is stated in the Medication Process v09." />

```

Comment:
I think it's better to use a proper change request for this

Reply:
See https://github.com/Nictiz/Nictiz-STU3-Zib2017/pull/307